ci/scripts/r_windows_build.sh

```diff
@@ -34,6 +34,12 @@ if [ "$RTOOLS_VERSION" = "35" ]; then
   # lib-4.9.3 is for libraries compiled with gcc 4.9 (Rtools 3.5)
   RWINLIB_LIB_DIR="lib-4.9.3"
 else
+  # Uncomment L38-41 if you're testing a new rtools dependency that hasn't yet sync'd to CRAN
```

Comment:
@jeroen have you updated the CRAN repo with aws-sdk-cpp yet?

Reply:
No not yet. Will do soon if we confirm that it works (and you are sure you don't want to add more modules again :))

Reply:
I believe the macOS jemalloc install issue has been fixed now but I don't think I can easily confirm that on https://github.com/autobrew/homebrew-core/pull/19 because you'd need to `brew install --HEAD` to pull the fix, and the `brew test-bot` command doesn't support head. 

From the arrow perspective, builds are passing, so if you're satisfied that this is working, we can merge. And once we merge, no modules can get added to the cmake config without breaking the build, so there shouldn't be any more surprises.

Reply:
You can try to cherry-pick the patch in the homebrew formula like so: https://github.com/autobrew/homebrew-core/blob/adebb0d668a1625c2c28d01df145ab7376ef5aef/Formula/mariadb-connector-c.rb#L9-L12

Reply:
Cool, that worked: https://travis-ci.org/github/autobrew/homebrew-core/builds/719332651

Reply:
Ok @jeroen, LMK when the CRAN rtools-packages is updated and I'll switch off of the bintray. No problem if that can't be done easily/quickly, I can merge this and swap the repos later.

Reply:
I have uploaded it to my server, it should sync to the mirrors in a few hours.